Automattic/pocket-casts-android
# Bookmarks - Show on podcast page

## Description

This displays bookmarks on the podcast details page.

## Testing Instructions

1. Launch the app
2. Play an episode and add a few bookmarks
3. Go to corresponding `Podcast Details` page
4. ✅  Notice that tabs are displayed
5. Tap on bookmarks tab
6. ✅ Notice that bookmarks are displayed

P.S. I have not added any empty view/ search/ sorting to keep the PR small and to get validation on the direction. 

A few bigger diffs are due to indentation changes and other changes include supporting a new view holder for tabs and bookmarks.

## Screenshots or Screencast 

<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/6c833e0b-51c4-4df9-bee8-ca382a16d994"/>


## Checklist
 
#### I have tested any UI changes...
